Libraries/LibWeb/DOM/EditingHostManager.cpp

```diff
@@ -156,6 +156,22 @@ void EditingHostManager::decrement_cursor_position_to_previous_word(CollapseSele
         return;
     selection->move_offset_to_previous_word(collapse == CollapseSelection::Yes);
 }
+void EditingHostManager::increment_cursor_position_to_next_line(CollapseSelection collapse)
+{
+    // TODO: Improve this method
```

Comment:
Please use a `// FIXME` for this to describe what currently is wrong and what should be happening instead.